src/code/z_actor.c

```diff
@@ -1768,7 +1768,7 @@ u32 Actor_TextboxIsClosing(Actor* actor, PlayState* play) {
     }
 }
 
-s8 func_8002F368(PlayState* play) {
+s8 Player_GetExchangeItemId(PlayState* play) {
     Player* player = GET_PLAYER(play);
 
     return player->exchangeItemId;
```

Comment:
Im unsure if this should be named after player. I see it more as an Actor function, as their way of seeing the exchange item being offered to them. This is the main reason I havent named it so far.

Reply:
ok, done 👍🏼 e5c5c320f82c14d45a2d5796eb37bd5dd2f2894f

Reply:
Thanks! feel free to open a PR for this separately, and we can further discuss if it should be a "player" or "actor" function. Im not sure yet myself.

I'll do a review of en_go2 shortly.

Reply:
here it is then: https://github.com/zeldaret/oot/pull/2358